jekyll/lizards_rss.rb

```diff
@@ -10,26 +10,25 @@
 # The code is based on the JekyllImport::Importers::RSS code
 module JekyllImport
   module Importers
+    # Specific lizards.o.o RSS importer
     class LizardsRSS < Importer
       def self.specify_options(c)
-        c.option 'source', '--source NAME', 'The RSS file or URL to import'
+        c.option "source", "--source NAME", "The RSS file or URL to import"
       end
 
       def self.validate(options)
-        if options['source'].nil?
-          abort "Missing mandatory option --source."
-        end
+        abort "Missing mandatory option --source." if options["source"].nil?
       end
 
       def self.require_deps
-        JekyllImport.require_with_fallback(%w[
-          rss
-          rss/1.0
-          rss/2.0
-          open-uri
-          fileutils
-          safe_yaml
-        ])
+        JekyllImport.require_with_fallback(%w(
```

Comment:
What about moving the `%w(` to the line below? Just to avoid that hug indentation.